src/kex_ntru/kex_ntru.c

```diff
@@ -78,10 +78,13 @@ int OQS_KEX_ntru_alice_0(UNUSED OQS_KEX *k, void **alice_priv, uint8_t **alice_m
 	DRBG_HANDLE drbg;
 	OQS_KEX_ntru_alice_priv *ntru_alice_priv = NULL;
 
+	*alice_priv = NULL;
+	*alice_msg = NULL;
+
 	/* initialize NTRU DRBG */
 	rc = ntru_crypto_drbg_instantiate(256, (uint8_t *) "OQS Alice", strlen("OQS Alice"), (ENTROPY_FN) &get_entropy_from_dev_urandom, &drbg);
```

Comment:
NTRU doesn't define a value to represent an invalid DRBG_HANDLE. Thus, `ntru_crypto_drbg_uninstantiate()` can only safely be called if `ntru_crypto_drbg_instantiate()` was successful.